GameServer/Logic/GameObjects/LaneTurret.cs

```diff
@@ -24,7 +25,7 @@ public class LaneTurret : BaseTurret
         private const float TURRET_RANGE = 905.0f;
         public TurretType Type { get; private set; }
         private bool _turretHPUpdated = false;
-        private int[] _items = new int[5];
```

Comment:
If you have an inventory, you don't need to list the items here as well. Wasted memory.